internal/tableland/tableland.go

```diff
@@ -41,8 +41,10 @@ type TxnReceipt struct {
 	ChainID     ChainID `json:"chain_id"`
 	TxnHash     string  `json:"txn_hash"`
 	BlockNumber int64   `json:"block_number"`
-	Error       *string `json:"error,omitempty"`
-	TableID     *string `json:"table_id,omitempty"`
+
+	TableID       *string `json:"table_id,omitempty"`
+	Error         *string `json:"error,omitempty"`
+	ErrorEventIdx int     `json:"error_event_idx,omitempty"`
```

Comment:
Definitely agree on removing `omitempty` for `ErrorEventIdx`.

For `Error`, agree on that too but I wonder if this may affect some behavior in the SDKs if they already have an interpretation between "not there" and "there but empty". I'll ping @joewagner . If all is fine, I'd also remove the `omitempty` in this field too.

Reply:
I talked with @joewagner  and have his :+1:. So, I'll also remove the `omitempty`  from `Error`.

Reply:
This looks like it works with the current SDK, so we should be all good having the "there but empty" error